### PR TITLE
Cleanup and updates

### PR DESCRIPTION
**Pull request by gabriel-samfira**
* Add db-prefix config option (readonly - set on initial charm
deploy)
  * Enable volume cache in cinder-volume
  * Add Ussuri support
  * Add TLS support via certificates relation
  * Add proper Keystone credentials relation

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -128,3 +134,18 @@ options:
...
+  enable-image-volume-cache:
+    type: boolean
+    default: true
+    description: |
+      When enabled, cinder will cache glance images locally.
+  image-volume-cache-size:
+    type: int
+    default: 100
+    description: |
+      The size in GB of the cache for the backend.
```
Comment (ionutbalutoiu) [inline config.yaml]: small nit: missing newline at the end of file

File: lib/Modules/CinderHooks/CinderHooks.psm1 [hunk 2]
```diff
@@ -283,11 +347,88 @@ function Get-CharmConfigContext {
...
+
+    if ($ctxt["ca_cert"]) {
+        Write-FileFromBase64 -File $ksCa -Content $ctxt["ca_cert"]
+        $ctxt["ks_ssl_ca_cert"] = $ksCa
+    }
+
+    return $ctxt
+}
+
 function Get-CloudComputeContext {
```
Comment (ionutbalutoiu) [inline lib/Modules/CinderHooks/CinderHooks.psm1]: We don't have the `cloud-compute` provides relation anymore.

Do we still need this function ?

**Review by ionutbalutoiu — APPROVED** [review]
Only had two comments. All good in rest!

File: lib/Modules/CinderHooks/CinderHooks.psm1 (hunk 2 above)
Comment (gabriel-samfira) [reply lib/Modules/CinderHooks/CinderHooks.psm1]: nope!

**Pull request merged**